go.sum

```diff
@@ -195,6 +195,7 @@ github.com/dgryski/go-farm v0.0.0-20190423205320-6a90982ecee2/go.mod h1:SqUrOPUn
 github.com/dgryski/go-sip13 v0.0.0-20181026042036-e10d5fee7954/go.mod h1:vAd38F8PWV+bWy6jNmig1y/TA+kYO4g3RSRF0IAv0no=
 github.com/dlclark/regexp2 v1.2.0/go.mod h1:2pZnwuY/m+8K6iRw6wQdMtk+rH5tNGR1i55kozfMjCc=
 github.com/docker/docker v1.4.2-0.20180625184442-8e610b2b55bf/go.mod h1:eEKB0N0r5NX/I1kEveEz05bcu8tLC/8azJZsviup8Sk=
+github.com/docker/go-healthcheck v0.1.0 h1:6ZrRr63F5LLsPwSlbZgjgoxNu+o1VlMIhCQWgbfrgU0=
```

Comment:
Is this package being used in this PR, looks like it opens 5001 by default? Just looking at it, it'd be nice to use some of its features, for example adding a check that if the node is active, it's still connected to the ETH client.

Reply:
Hey @jleeh, thanks for the review!
To be honest with you, I am not a Go developer, so I only implemented the basic use case described in the package's documentation. But I totally agree, this health check could be much smarter.

As for the opening of door 5001, I didn't see that. But unless you use the `EXPOSE` tag in the Dockerfile this port would never see the light of the day.